raster/apply-mosaic-rule-to-rasters/build.gradle

```
@@ -6,12 +6,17 @@ plugins {
 group = 'com.esri.samples'
 
 ext {
-    arcgisVersion = '100.15.0'
+    arcgisVersion = '200.0.0-3634'
 }
 
 javafx {
     version = "17.0.2"
-    modules = [ 'javafx.controls', 'javafx.fxml']
+    modules = [ 'javafx.controls', 'javafx.media', 'javafx.fxml', 'javafx.web']
```

```suggestion
    modules = [  'javafx.base', 'javafx.graphics', 'javafx.controls', 'javafx.fxml', 'javafx.web', 'javafx.media' ]
```
Initially we'll declare all.